nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -2129,44 +2127,6 @@ pub mod test {
         assert_eq!(diff.sleds_modified.len(), 0);
     }
 
-    #[test]
-    fn test_initial() {
```

Comment:
The only thing this test does is diff a blueprint and a collection, so it had to go.